File: Bars.GkhGJI/libs/B4/model/Dict/ControlList.js
Ext.define('B4.model.dict.ControlList', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'ControlList'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Name' },
        { name: 'Code' },
        { name: 'KindKNDGJI', defaultValue: 0 },
        { name: 'ActualFrom' },
        { name: 'ERKNMGuid' },
        { name: 'ActualTo' }
    ]
});

File: Bars.GkhGJI/Entities/Dict/ControlList.cs
namespace Bars.GkhGji.Entities.Dict
{
    using Bars.Gkh.Entities;
    using Bars.GkhGji.Enums;
    using System;

    public class ControlList : BaseGkhEntity
    {
        /// <summary>
        /// Код
        /// </summary>
        public virtual string Code { get; set; }

        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Вид контроля(надзора)
        /// </summary>
        public virtual KindKNDGJI KindKNDGJI { get; set; }


        /// <summary>
        /// Дата актуальности с
        /// </summary>
        public virtual DateTime ActualFrom { get; set; }

        /// <summary>
        /// Дата актуальности по
        /// </summary>
        public virtual DateTime? ActualTo { get; set; }

        /// <summary>
        /// ЕРКНМ ГУИД
        /// </summary>
        public virtual string ERKNMGuid { get; set; }

    }
}
